cmd/minikube/cmd/start_flags.go

```diff
@@ -293,7 +306,9 @@ func initNetworkingFlags() {
 	startCmd.Flags().StringSliceVar(&registryMirror, "registry-mirror", nil, "Registry mirrors to pass to the Docker daemon")
 	startCmd.Flags().String(imageRepository, "", "Alternative image repository to pull docker images from. This can be used when you have limited access to gcr.io. Set it to \"auto\" to let minikube decide one for you. For Chinese mainland users, you may use local gcr.io mirrors such as registry.cn-hangzhou.aliyuncs.com/google_containers")
 	startCmd.Flags().String(imageMirrorCountry, "", "Country code of the image mirror to be used. Leave empty to use the global one. For Chinese mainland users, set it to cn.")
-	startCmd.Flags().String(serviceCIDR, constants.DefaultServiceCIDR, "The CIDR to be used for service cluster IPs.")
+	startCmd.Flags().String(serviceCIDR, constants.DefaultServiceCIDR, "The IPv4 CIDR to be used for service cluster IPs.")
+	startCmd.Flags().String(serviceCIDRv6, constants.DefaultServiceCIDRv6, "The IPv6 CIDR to be used for service cluster IPs.")
```

Comment:
This means --service-cluster-ip-range-v6 is always set (via the flag default), even when the user runs IPv4-only.

Later you check for "" not the default:
```go
	if fam != "ipv4" {
		if cc.KubernetesConfig.ServiceCIDRv6 == "" {
			cc.KubernetesConfig.ServiceCIDRv6 = constants.DefaultServiceCIDRv6
		}
		if cc.KubernetesConfig.PodCIDRv6 == "" {
			cc.KubernetesConfig.PodCIDRv6 = constants.DefaultPodCIDRv6
		}
	}
```


So I think these should be instead with "":
```go
    startCmd.Flags().String(serviceCIDR, "", "The IPv4 CIDR to be used for service cluster IPs.")
    startCmd.Flags().String(serviceCIDRv6, "", "The IPv6 CIDR to be used for service cluster IPs (defaults to fd00::/108 when --ip-family=ipv6|dual).")
```